### PR TITLE
chall03

### PR DESCRIPTION
**Pull request by osalmine**
First I get the values from the website, then I parse the values out of the string I got from the website, convert the rgb values to hex and get the response from the website with the id and the hex value.

### CODE REVIEW

File: chall03/osalmine.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+func get_rgb_id (s string) (int, int, int, int) {
+	str := strings.Split(s, " ")
+	vals := strings.Split(str[0], ",")
+	id, err := strconv.Atoi(strings.Split(vals[0], "=")[1]);
+	r, err := strconv.Atoi(strings.Split(vals[1], "=")[1]);
+	g, err := strconv.Atoi(strings.Split(vals[2], "=")[1]);
+	b, err := strconv.Atoi(strings.Split(vals[3], "=")[1]);
+	if err != nil {
+		log.Fatal(err)
+	}
```
Comment (jgengo) [inline chall03/osalmine.go:21]: You should check `error != nil` right after each those lines.

File: chall03/osalmine.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	resp, err := http.Get(url)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatal(err)
+	}
+	s := fmt.Sprintf("%s", body)
```
Comment (jgengo) [inline chall03/osalmine.go:36]: Can you explain why you did that? I don't really understand the purpose

**Review by jgengo — APPROVED** [review]

File: chall03/osalmine.go (hunk 2 above)
Comment (osalmine) [reply chall03/osalmine.go:36]: If you mean `s := fmt.Sprintf("%s", body)`, I was just converting the body that was a list of decimals I believe into a string. The only way I knew how to do this was to use sprintf. There is probably a better way of doing this but this was my first go code 😅

Comment (jgengo) [reply chall03/osalmine.go:36]: You did a pretty good job.

I think `s := string(body)` is what you are looking for.

**Pull request merged**